### ISSUE TITLE
0.4.4 Client Crash - SymbolSunTinted$CelestialObject.render

### ISSUE THREAD

**Issue by Odin-sama** [TheFloydman/More-Mystcraft#11]
**What version of *More Mystcraft* are you using?**
0.4.4

**What version of *Mystcraft* are you using?**
0.13.7.02

**What version of *Forge* are you using?**
14.23.5.2806

**What version of *Minecraft* are you using?**
1.12.2

**What modpack and modpack version are you using, if any?**
N/A (Custom)

**Does the bug occur server-side, client-side, or both?**
Client-side

**Paste a link to your log or crash report here. Use a service like pastebin. No images, please.**
https://pastebin.com/w81DjdZT

**Describe the bug**
I've loaded a world without specification as to Sun parameters, save for 'Zenith Phase' expected to attach to it as a loose modifier. Tinted Sun nor any colors were specified from a page. Upon loading this world, client crashes with null pointer exception at More Mystcraft, appearing to have something to do with the way the mod handles Tinted Sun rendering. Maybe an invalid color attempting to render? 

**To Reproduce**
1. [Generate world - Age parameters for sun were random]
2. Client crashes upon loading new age. The dimension server-side was however, successfully created.

**Expected behavior**
Load into world as expected without a crash.

**Additional Information**
Server appears to generate no errors upon client crash besides reporting that a client timed out. The age appears to have successfully generated a DIM-MYST## folder and looks like a valid world. I can provide the Dimension folder in addition to a full world save and our custom mod pack to test with, if desired.

**Comment by TheFloydman** [comment]
Thank you for finding this. It appears I neglected to a null check on one of the variables in the Tinted Sun. I have updated it so that if it's null, it defaults to a yellow color, which should fix the problem. I know this is unlikely, but have you disabled the color pages in Mystcraft's _symbols.cfg_?

**Comment by Odin-sama** [comment]
Awesome response, and thanks for the quick fix. I've loaded it to my server and tested it in the client; The world loaded successfully! Looks like a white-ish sun generated.
![2019-01-16_19 11 40](https://user-images.githubusercontent.com/46764217/51288635-b145c400-19c2-11e9-8162-41c3fd351c41.jpg)

I did not disable colors in my symbols.cfg, no. I double-checked the file searching anything 'color' and it's all flagged for true.

I also had this pop up upon loading the server. Figured I'd let you know in case this happened to be unintentional:

_Missing minecraft:items:
    moremystcraft:maintainer_suit
_
